airflow/www/static/js/dag/Main.tsx

```diff
@@ -65,24 +70,83 @@ const Main = () => {
     onToggle();
   };
 
+  useEffect(() => {
+    if (contentRef.current) {
+      const topOffset = contentRef.current.offsetTop;
+      const footerHeight = parseInt(getComputedStyle(document.getElementsByTagName('body')[0]).paddingBottom.replace('px', '').replace('em', ''), 10) || 0;
```

Comment:
This feels weird to replace indifferently `px` and `em`. I think footerHeight won't  have the correct value if it is specified in 'em'. (We use it as a pixel value later in the code).